bootstrap/templates/k0s-config.yaml.j2

```diff
@@ -0,0 +1,118 @@
+#jinja2: trim_blocks: True, lstrip_blocks: True
```

Comment:
Across this file I would keep the YAML consistency and indent arrays. e.g. instead of

```yaml
  hosts:
  {% for item in bootstrap_nodes.master %}
  {% if (item.role is not defined) and (bootstrap_nodes.master | length > 1) %}
  - role: controller+worker
```

use

```yaml
  hosts:
  {% for item in bootstrap_nodes.master %}
  {% if (item.role is not defined) and (bootstrap_nodes.master | length > 1) %}
    - role: controller+worker
```